pandasai/prompts/rephase_query_prompt.py

```diff
@@ -0,0 +1,45 @@
+""" Prompt to rephrase query to get more accurate responses
+You are provided with the following pandas DataFrames:
+
+{dataframes}
+
+and based on our conversation:
+
+<conversation>
+{conversation}
+</conversation>
+
+Return the rephrased sentence of  "{query}”  in order to obtain more accurate and 
+comprehensive responses without any explanations.
+
+"""
+from typing import List
+
+import pandas as pd
+from .base import Prompt
+
+
+class RephraseQueryPrompt(Prompt):
+    """Prompt to rephrase query to get more accurate responses"""
+
+    text: str = """
+You are provided with the following pandas DataFrames:
+
+{dataframes}
+
+And based on our conversation:
+
+<conversation>
+{conversation}
+</conversation>
+
+Return the rephrased sentence of "{query}” in order to obtain more accurate and 
+comprehensive responses without any explanations. If something from the original
+query is ambiguous, please clarify it in the rephrased query, making assumptions,
+if necessary.
```

Comment:
@ArslanSaleem just FYI, I've slightly changed the prompt to also make assumptions and including them in the question.